packages/flow-core/src/components/f-tag/f-tag.test.ts

```diff
@@ -0,0 +1,90 @@
+import { html, fixture, expect } from "@open-wc/testing";
+import IconPack from "@cldcvr/flow-system-icon/dist/types/icon-pack";
+
+// import flow-core elements
+import "@cldcvr/flow-core";
+
+import { FTag, FIcon, ConfigUtil, FCounter } from "@cldcvr/flow-core";
+// importing `loadingSVG` to cross check
+import loadingSVG from "../../mixins/svg/loader";
+
+// setting icon pack for testing icon related test cases
+ConfigUtil.setConfig({ iconPack: IconPack });
+
+describe("f-tag", () => {
+  it("is defined", () => {
+    const el = document.createElement("f-tag");
+    expect(el).instanceOf(FTag);
+  });
+  it("should render label in default slot", async () => {
+    const el = await fixture(html` <f-tag label="abc" state="neutral"></f-tag> `);
+    const descendant = el.shadowRoot!.querySelector(".f-tag")!;
+    expect(descendant.textContent?.trim()).to.equal("abc");
+  });
+  it("should throw error", async () => {
+    try {
+      await fixture(html` <f-tag state="neutral"></f-tag>`);
+    } catch (e) {
+      expect((e as Error).message).to.equal("f-tag : label OR icon-left is mandatory field");
+    }
+  });
+  it("should render neutral state", async () => {
+    const el = await fixture(html` <f-tag label="abc" state="neutral"></f-tag> `);
+    const descendant = el.shadowRoot!.querySelector(".f-tag")!;
+    expect(descendant.getAttribute("state")).to.equal("neutral");
```

Comment:
what neutral suppose to change in the component, please test that. you can use `getComputedStyle ` if required.